### PR TITLE
Update UG UndoCommand

### PR DESCRIPTION
**Pull request by glenngnng**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -200,29 +200,99 @@ Precise expected outputs on failure:
...
+
+For undoing a delete command:
 
-![edit format](images/undo-UG/one_contact_GUI.png)
+* Delete a contact.
 
 ![edit format](images/undo-UG/after_delete_1.png)
 
-Then, undo the delete.
+* Undo .
```
Comment (syamfarh) [inline docs/UserGuide.md]: nitpick: a space before a period

**Review by syamfarh — COMMENTED** [review]
LGTM, one nitpick

File: docs/UserGuide.md (hunk 1 above)
Comment (glenngnng) [reply docs/UserGuide.md]: Resolved!

**Comment by syamfarh** [comment]
LGTM!

**Pull request merged**